docs/bok/Activities/Level-3/Contribution-Training.md

```diff
@@ -153,7 +153,9 @@ This has a large impact on the shape of the resulting policy and therefore also
 
  - Certain types of open source contribution might represent a conflict of interest. 
  
-  tbd, traing requirement around this.  
+  tbd, traing requirement around this.
```

Comment:
Should you make this tbd somehow differently?

Reply:
should say "training".   wasn't sure how to substantiate this - any thoughts?

Reply:
As a start can link to https://www.linuxfoundation.org/legal/antitrust-policy here